<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Init</class>
 <widget class="QWidget" name="Tab_Init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Composition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2" colspan="3">
       <widget class="QLineEdit" name="edit_composition"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Cell composition:</string>
        </property>
        <property name="buddy">
         <cstring>edit_composition</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QTableWidget" name="table_composition">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Atom</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Constrain</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>x (frac)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y (frac)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z (frac)</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Unit Cell Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Length B (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_b_min</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Length C (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_c_min</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Angle α (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_alpha_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Length A (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_a_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spin_a_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spin_b_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spin_c_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spin_alpha_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="spin_a_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="spin_b_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="spin_c_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="spin_alpha_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Minimum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Angle β (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_beta_min</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Volume (Å&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>spin_vol_min</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spin_beta_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spin_gamma_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="spin_vol_min">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="spin_beta_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QDoubleSpinBox" name="spin_vol_max">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="spin_gamma_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Angle γ (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_gamma_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QDoubleSpinBox" name="spin_fixedVolume">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use this to specify a starting volume for the structure before optimizing.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QCheckBox" name="cb_fixedVolume">
        <property name="toolTip">
         <string>Use this to specify a starting volume for the structure before optimizing.</string>
        </property>
        <property name="text">
         <string>Fixed volume:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QCheckBox" name="cb_shortestInteratomicDistance">
        <property name="toolTip">
         <string>Use this to limit the shortest interatomic distance between atoms. This ensures that atoms aren't overlapping.</string>
        </property>
        <property name="text">
         <string>Limit interatomic distance? (Å)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QDoubleSpinBox" name="spin_shortestInteratomicDistance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use this to limit the shortest interatomic distance between atoms. This ensures that atoms aren't overlapping.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_composition</tabstop>
  <tabstop>table_composition</tabstop>
  <tabstop>spin_a_min</tabstop>
  <tabstop>spin_a_max</tabstop>
  <tabstop>spin_b_min</tabstop>
  <tabstop>spin_b_max</tabstop>
  <tabstop>spin_c_min</tabstop>
  <tabstop>spin_c_max</tabstop>
  <tabstop>spin_alpha_min</tabstop>
  <tabstop>spin_alpha_max</tabstop>
  <tabstop>spin_beta_min</tabstop>
  <tabstop>spin_beta_max</tabstop>
  <tabstop>spin_gamma_min</tabstop>
  <tabstop>spin_gamma_max</tabstop>
  <tabstop>spin_vol_min</tabstop>
  <tabstop>spin_vol_max</tabstop>
  <tabstop>cb_fixedVolume</tabstop>
  <tabstop>spin_fixedVolume</tabstop>
  <tabstop>cb_shortestInteratomicDistance</tabstop>
  <tabstop>spin_shortestInteratomicDistance</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_shortestInteratomicDistance</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_shortestInteratomicDistance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_fixedVolume</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_vol_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_vol_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
